client/src/Components/Header.jsx

```diff
@@ -1,15 +1,36 @@
 import { AppBar, Box, Button } from "@mui/material";
 import React from "react";
+import { useNavigate } from "react-router-dom";
+import useAuth from "../customHooks/useAuth";
 
 const Header = () => {
+	const navigate = useNavigate();
+  const { user, handleSignUp } = useAuth();
+
   return (
-		<AppBar sx={{ background: "white" }}>
-			<Box sx={{ display: "flex", justifyContent: "flex-end", py: 2, px: 5, gap: 5 }}>
-				<Button variant="outlined">Sign Up</Button>
-				<Button variant="contained" >Log in</Button>
-			</Box>
-		</AppBar>
-	);
+    <AppBar sx={{ background: "white" }}>
+      <Box sx={{ display: "flex", justifyContent: "flex-end", py: 2, px: 5, gap: 5 }}>
+        { !user ? (
+          <Box>
+            <Button variant="outlined" onClick={handleSignUp}>
+              Sign Up
+            </Button>
+            <Button variant="outlined" onClick={handleSignUp}>
+              Sign In
+            </Button>
+          </Box>
+        ) : (
+          <></>
+        )}
```

Comment:
Can we try this? Not sure if that will work but would be consistent with the approach below. For the team to discuss
`
{ !user && (
          <Box>
            <Button variant="outlined" onClick={handleSignUp}>
              Sign Up
            </Button>
            <Button variant="outlined" onClick={handleSignUp}>
              Sign In
            </Button>
          </Box>
        ) 
`